README.md

```diff
@@ -1,3 +1,7 @@
 # FirstContribution2
 
 Testing PR comments
+
+Before you submit your review, your line comments are pending and only visible to you. You can edit pending comments anytime before you submit your review. To cancel a pending review, including all of its pending comments, click Review changes above the changed code, then click Abandon review.
```

Comment:
First comment on Files Changed tab with no review started.

Reply:
Replying to comment that was not marked as pending since it was done before review started.

Reply:
Stil pending?